install/install.go

```diff
@@ -35,19 +37,63 @@ type Options struct {
 	Save           bool
 }
 
+type progressBar struct {
+	bar string
+	fill string
+	blank string
+	filled int
+	unfilled int
+	step int
+	done float64
+	size float64
+}
+
+var progbar progressBar
+
+func printMessage(msg string,prog bool) int {
+	if progbar.filled == 0 {
+		fmt.Printf("KARMOR-CLI INSTALL \n \n")
```

Comment:
fmt.Printf("Install KubeArmor \n \n")

Reply:
I think we should remove this print altogether. User has already specified `karmor install` on the command line just a nanosecond before :-)

Reply:
Okay we discussed that we should remove this will do so ...